myuw/templates/handlebars/card/accounts/husky.html

```diff
@@ -7,8 +7,24 @@ <h3 aria-hidden="true">Husky Card</h3>
     <h3 class="sr-only">Husky Card accounts</h3>
 
     <div class="card-badge-container">
-	<ul class="card_list">
-            {{#if student_husky_card}}
+    <!--Tacoma Dawg Dollars    -->
+    {{#if is_tacoma}}
+    <ul class="card_list" style="margin-bottom: 0px;">
+        <li>
+            <div class="card-badge clearfix">
+                <div class="pull-left">
+                    <h4 class="card-badge-label" aria-hidden="true">Tacoma Dawg Dollars</h4>
+                    <h4 class="sr-only">Tacoma Dawg Dollars Account</h4>
+                </div>
+                <div class="pull-right">
+                    <div class="btn btn-default card-action-btn" ><a href="http://apps.tacoma.uw.edu/dawgdollars/" target="_blank" data-linklabel="Tacoma Dawg Doallar account">Manage account</a></div>
+                </div>
+            </div>
+        </li>
+     </ul>
```

Comment:
Not a release stopper, necessarily, but I think this card should just have a single `<ul>`. It is a little strange to have a "list" with a single item in it (tacoma dawg dollars), and then to follow it with another, separate list with the Husky Card balance information in it. It looks like the list items are used elsewhere to enumerate whatever is labelled in the card. 

A single list would be more semantic and better for accessibility.

Reply:
Changed it to single <ui> for the Dawg Dollars account and Husky accounts.